pages/linux/nixos-rebuild.md

```diff
@@ -30,3 +30,7 @@
 - Build the configuration and open it in a virtual machine:
 
 `sudo nixos-rebuild build-vm`
+
+- List available generations similar to the boot loader menu:
+
+`nixos-rebuild list-generations {{[--json]}}`
```

Comment:
```suggestion
`nixos-rebuild list-generations --json`
```
Not the proper usage. This is meant for when an option has longform and shortform variations.